yeast-markdown-parser/src/plugins/inline/InlineEmphasisPlugin.ts

```diff
@@ -0,0 +1,100 @@
+import {
+	InlineTokenizerPlugin,
+	Token,
+	YeastParser,
+	YeastNodeFactory,
+	YeastInlineNode,
+	YeastInlineNodeTypes,
+	YeastInlineChild,
+} from 'yeast-core';
+
+const ITALICS_REGEX_UNDERSCORES = /(\\?)(_)([^\s_]|\S.*?\S)(\\?)(_)/gi;
+const ITALICS_REGEX_ASTERISKS = /(\\?)(\*)([^\s_]|\S.*?\S)(\\?)(\*)/gi;
```

Comment:
I just found an edge case that this isn't handling (a single/unpaired escape within a match). Need to make an update here.